### PR TITLE
Create LiquidID.cs

### PR DESCRIPTION
**Pull request by RighteousRyan1**
### What is the new feature?
LiquidID

### Why should this be part of tModLoader?
To make reading Liquid Code and such easier on the eye.

### Are there alternative designs?
No.

### Sample usage for the new feature
`liquidType == LiquidID.Water;`

### ExampleMod updates
N/A

### CODE REVIEW

**Comment by direwolf420** [comment]
This already exists in the game:
```cs
public class Tile
{
    public const int Type_Solid = 0;
    public const int Liquid_Honey = 2;
    public const int Liquid_Lava = 1;
    public const int Liquid_Water = 0;

    //...
}
```

Used like: `if (tile.liquidType() == Tile.Liquid_Honey)`

**Comment by Chicken-Bones** [comment]
There you have it, but also oof to determining tile ids in reverse

File: patches/tModLoader/Terraria/ID/LiquidID.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace Terraria.ID
+{
+  /// <summary> Enumerates liquid values to make them easier to see/read </summary>
+  public static class LiquidID
```
Comment (Mirsario) [inline patches/tModLoader/Terraria/ID/LiquidID.cs:4]: Tabs pls

Comment (RyanMakesMods) [reply patches/tModLoader/Terraria/ID/LiquidID.cs]: done.

**Comment by Chicken-Bones** [comment]
are we accepting this? given that the fields exist in Tile?

**Comment by JavidPack** [comment]
I don't really see the point, I'm already using the Tile class consts in my mod, having two places for the same info isn't ideal.

**Comment by Mirsario** [comment]
Well, the consts in Tile can be made internal.

**Comment by Chicken-Bones** [comment]
/polls "Accept (make Tile.Liquid_* internal)" "Reject (modders just use Tile.Liquid_)"

**Comment by Chicken-Bones** [comment]
Deferred to #1301
I think there's a stronger case for it with the rest of the property rework. It's not something I actually have an opinion on

**Pull request closed without merging**